CdIts.Caffoa.Cli/Generator/ModelGenerator.cs

```diff
@@ -56,7 +56,8 @@ private void WriteModelClass(SchemaItem item, List<SchemaItem> interfaces)
         parameters["PROPERTIES"] = FormatProperties(item);
         parameters["DESCRIPTION"] = formatter.Description;
         var formatted = file.FormatDict(parameters);
-        File.WriteAllText(Path.Combine(_service.Model.TargetFolder, fileName), formatted);
+        formatted = formatted.Replace("\r\n", "\n").Replace("\n", Environment.NewLine);
```

Comment:
don't toSystemNewLine() do the same?

Reply:
yes, this was the first try, i forgot to remove it